### PR TITLE
Polish export/import for version 11

### PR DESCRIPTION
**Pull request by GermanDZ**
After loading a full import, now 3 extra steps are being performed:

- set the authentication flow to some clients (the version 11.x is not able to import the `authenticationFlowBindingOverrides`)
- set secrets to clients
- set secrets to identity providers

Also, the `export` now redacts the secrets in the output file

**Note**: Some commits that come from another PR, will be removed after #16 is merged.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ services:
   keycloak-config-cli:
     profiles:
       - config
-    image: adorsys/keycloak-config-cli:latest
+    image: adorsys/keycloak-config-cli:latest-11.0.3
```
Comment (BuonOmo) [inline docker-compose.yml:61]:
```suggestion
    image: adorsys/keycloak-config-cli:latest-${KEYCLOAK_VERSION}
```
But I've done that one in snapshots+ anyway :)

**Review by BuonOmo — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (GermanDZ) [reply docker-compose.yml:61]: 🤔 it's a little more complicated. We use `KEYCLOAK_VERSION = 11.0.0` (it's the version in production), but this tool is not working with that version, needs the fixed client in version `11.0.3`.

We need to be able to override the `KEYCLOAK_CONFIG_CLI_VERSION`, right now we are hardcoded into `11.0.0` and `11.0.3`.

Comment (BuonOmo) [reply docker-compose.yml]: couldn't we use the patched version in production ? It seems sensible anyway :)

Comment (GermanDZ) [reply docker-compose.yml]: I wish, I don't know how infra is selecting the versions. Is not easy/quick to perform those upgrades for them (they need to plan the upgrade) and they perform all the steps manually.

**Pull request merged**